### PR TITLE
Polish todo ux

### PR DESCRIPTION
**Pull request by greatbacon**
This PR makes an effort to implement the polished UX for the Todo view outlined here: https://mydraft.cc/c8keok7malqvgmbsvncg 

The "clear done" I think will be a follow up for after we get our store situation figured out.

### CODE REVIEW

File: src/lib/ui/EntityInput.svelte [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+		/>
+		{#if errorMessage}
+			<Message status="error">{errorMessage}</Message>
+		{/if}
+		<button type="button" on:click={create} disabled={status === 'pending'}> Create entity </button>
+	</form>
+</div>
+
+<style>
+</style>
```
Comment (ryanatkn) [inline src/lib/ui/EntityInput.svelte:78]: This looks great, will be fun to expand it for the generic usecases.

File: src/lib/ui/TodoItems.svelte [hunk 2]
```diff
@@ -4,17 +4,23 @@
...
 
 	export let entities: Readable<Array<Readable<Entity>>>;
 	//TODO properly wrap in reactive store
 	export let ties: Tie[];
 	export let itemsByEntity: Map<Readable<Entity>, Array<Readable<Entity>>>;
 	export let entityById: Map<number, Readable<Entity>>;
+	export let selectedList: Entity | null;
+	export let selectList: (list: Entity) => void;
+
+	//TODO in directory structure, this would just grab the "lists" collection from the dir
```
Comment (ryanatkn) [inline src/lib/ui/TodoItems.svelte:17]: # :+1:

**Review by ryanatkn — APPROVED** [review]
looks great!

**Comment by ryanatkn** [comment]
CI failing is something I've seen before, if you try re-running the job that's worked for me. Looking into it now.

File: src/lib/ui/TodoItem.svelte [hunk 3]
```diff
@@ -88,22 +80,21 @@ And then PersonaContextmenu would be only for *session* personas? `SessionPerson
 				<Avatar name={toName($persona)} icon={toIcon($persona)} />
 			</div>
 			<div>
-				{$entity.data.content}::{$entity.entity_id}
+				{$entity.data.content || $entity.data.name}
```
[the remark this answers is not in the document]
Comment (greatbacon) [reply src/lib/ui/TodoItem.svelte:83]: That works for me!

**Pull request merged**